test/test_ea_mu_plus_lambda.py

```diff
@@ -57,3 +57,97 @@ def objective(ind):
 
     for idx, ind in enumerate(offsprings):
         assert ind.idx == len(pop.parents) + idx
+
+
+def test_n_offspring_less_than_n_breeding():
+    n_offsprings = 10
+    n_breedings = 5
+    with pytest.raises(ValueError):
+        _ = cgp.ea.MuPlusLambda(n_offsprings, n_breedings, 1)
+
+
+def test_fitness_wrong_return_type(population_params, genome_params):
+    def objective(individual):
+        individual.fitness = 5
```

Comment:
```suggestion
        individual.fitness = int(5.0)  # should raise error since fitness should be float
```
just to be super explicit